app/src/main/java/com/github/globant/githubsubscribers/subscriberslist/view/SubscriberItem.java

```diff
@@ -0,0 +1,48 @@
+package com.github.globant.githubsubscribers.subscriberslist.view;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.ImageView;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.github.globant.githubsubscribers.R;
+import com.github.globant.githubsubscribers.commons.models.Subscriber;
+import com.squareup.picasso.Picasso;
+
+/**
+ * @author edwin.cobos
+ * @since 30/08/2016
+ */
+public class SubscriberItem extends RelativeLayout {
```

Comment:
I think use this custom layout to separate this from the adapter.
This class is used in the Adapter class.